src/rabbit_stomp_processor.erl

```diff
@@ -408,7 +408,7 @@ ack_action(Command, Frame,
                 {ok, {ConsumerTag, _SessionId, DeliveryTag}} ->
                     case maps:find(ConsumerTag, Subs) of
                         {ok, Sub} ->
-                            Requeue = rabbit_stomp_frame:boolean_header(Frame, "requeue", true),
+                            Requeue = rabbit_stomp_frame:boolean_header(Frame, "requeue", application:get_env(rabbitmq_stomp, default_nack_requeue, true)),
```

Comment:
All calls to `application:get_env/{2,3}` go through a single process. Never ever call it on a hot code path.

Reply:
@michaelklishin - appreciate the swift response and action. With my erlang knowledge being limited, would you be kind to pin-point the existing example on how this is currently being done for some other config options? I have based my approach on the `hide_server_info` [case](https://github.com/rabbitmq/rabbitmq-stomp/blob/master/src/rabbit_stomp_processor.erl#L609).

Appreciate the time and assitance

Reply:
Read the value and store it in process state.

Reply:
Done, thanks

Reply:
@michaelklishin 

Should we also (in a future PR perhaps) also refactor:
- https://github.com/rabbitmq/rabbitmq-stomp/blob/master/src/rabbit_stomp_processor.erl#L278
-https://github.com/rabbitmq/rabbitmq-stomp/blob/master/src/rabbit_stomp_processor.erl#L609
To be recorded and used from the process state? Or, those have different behaviour or reason to not be using state?

Reply:
See what those functions handle. They handle `CONNECT` frames and authentication, which only happen once per connection.